config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,11 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider:              'AWS',
+    aws_access_key_id:     Rails.application.secrets.aws_access_key_id,
+    aws_secret_access_key: Rails.application.secrets.aws_secret_access_key,
+    region:                'us-east-1'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.